README.md

```diff
@@ -2,55 +2,16 @@
 
 [![CI](https://github.com/maplibre/martin/workflows/CI/badge.svg)](https://github.com/maplibre/martin/actions)
 ![Security audit](https://github.com/maplibre/martin/workflows/Security%20audit/badge.svg)
-
+[![Slack chat](https://img.shields.io/badge/Chat-on%20Slack-brightgreen)](https://slack.openstreetmap.us/)  
 Martin is a tile server able to generate [vector tiles](https://github.com/mapbox/vector-tile-spec) from large [PostGIS](https://github.com/postgis/postgis) databases on the fly, or serve tiles from [PMTile](https://protomaps.com/blog/pmtiles-v3-whats-new) and [MBTile](https://github.com/mapbox/mbtiles-spec) files. Martin optimizes for speed and heavy traffic, and is written in [Rust](https://github.com/rust-lang/rust).
 
 ![Martin](https://raw.githubusercontent.com/maplibre/martin/main/logo.png)
 
-<!-- TOC -->
-* [Requirements](#requirements)
-* [Installation](#installation)
-* [Usage](#usage)
-* [API](#api)
-* [Using with MapLibre](#using-with-maplibre)
-* [Using with Leaflet](#using-with-leaflet)
-* [Using with deck.gl](#using-with-deckgl)
-* [Using with Mapbox](#using-with-mapbox)
-* [Source List](#source-list)
-* [Composite Sources](#composite-sources)
-  * [Composite Source TileJSON](#composite-source-tilejson)
-  * [Composite Source Tiles](#composite-source-tiles)
-* [Table Sources](#table-sources)
-  * [Table Source TileJSON](#table-source-tilejson)
-  * [Table Source Tiles](#table-source-tiles)
-* [Function Sources](#function-sources)
-  * [Function Source TileJSON](#function-source-tilejson)
-  * [Function Source Tiles](#function-source-tiles)
-* [MBTile and PMTile Sources](#mbtile-and-pmtile-sources)
-* [Command-line Interface](#command-line-interface)
-* [Environment Variables](#environment-variables)
-* [Configuration File](#configuration-file)
-* [PostgreSQL Connection String](#postgresql-connection-string)
-  * [PostgreSQL SSL Connections](#postgresql-ssl-connections)
-* [Using with Docker](#using-with-docker)
-* [Using with Docker Compose](#using-with-docker-compose)
-* [Using with Nginx](#using-with-nginx)
-  * [Rewriting URLs](#rewriting-urls)
-  * [Caching tiles](#caching-tiles)
-* [Building from Source](#building-from-source)
-* [Debugging](#debugging)
-* [Development](#development)
-  * [Other useful commands](#other-useful-commands)
-* [Recipes](#recipes)
-  * [Using with DigitalOcean PostgreSQL](#using-with-digitalocean-postgresql)
-  * [Using with Heroku PostgreSQL](#using-with-heroku-postgresql)
-<!-- TOC -->
-
-# Requirements
+## Requirements
 
 Martin requires PostGIS 3.0+.  PostGIS 3.1+ is recommended.
```

Comment:
this should probably be corrected -- PostGIS is only needed when using Martin with PostgreSQL, not for mbtiles/pmtiles

Reply:
Description updated